File: Exercícios/Python Mundo 1/Desafio_18.py
# Exercício Python 18: Faça um programa que leia um ângulo qualquer e mostre na tela o valor do seno, cosseno e tangente desse ângulo.

import math
import numpy as np

print('=' * 6 + ' DESAFIO 18 ' + '=' * 6)

angulo = int(input('[?] - DIGITE UM ÂNGULO: '))
seno = math.sin(math.radians(angulo))
cosseno = math.cos(math.radians(angulo))
tangente = math.tan(math.radians(angulo))

print(f'[!] - SENO = [{seno:.2f}]')
print(f'[!] - COSSENO = [{cosseno:.2f}]')
print(f'[!] - TANGENTE - [{tangente:.2f}]')

File: Exercícios/Scripts_C#/ScriptsUsandoC#/ScriptsUsandoC#/Desafio_18.cs
class Desafio_18
{
    public static void SenCosTan()
    {
        Console.Write("INFORME UM ÂNGULO: ");
        double angulo = double.Parse(Console.ReadLine().Trim());
        double anguloRadiano = angulo * Math.PI / 180;

        double seno = Math.Sin(anguloRadiano);
        double cosseno = Math.Cos(anguloRadiano);
        double tangente = Math.Tan(anguloRadiano);

        Console.Write($"[SENO] = {seno}\n[COSSENO] - {cosseno}\n[TANGENTE] - {tangente}");
    }
}
